settings/OpenExceptionalForm/ExceptionWrapper.js

```diff
@@ -918,13 +907,26 @@ class ExceptionWrapper extends React.Component {
     }
   }
 
-  deleteException() {
+  deleteException = () => {
     const promises = [];
-    const { parentMutator } = this.props;
-    for (let i = 0; i < this.state.editor.exceptionalIds.length; i++) {
-      parentMutator.query.replace(this.state.editor.exceptionalIds[i].servicePointId);
-      parentMutator.periodId.replace(this.state.editor.exceptionalIds[i].id);
-      const a = this.props.parentMutator.periods.DELETE(this.state.editor.exceptionalIds[i].id);
+    const {
+      parentMutator: {
+        query,
+        periodId,
+        periods,
+      }
+    } = this.props;
+    const {
+      servicePoints,
+      editor: {
+        exceptionalIds,
+      }
+    } = this.state;
+
+    for (let i = 0; i < exceptionalIds.length; i++) {
+      query.replace(exceptionalIds[i].servicePointId);
+      periodId.replace(exceptionalIds[i].id);
+      const a = periods.DELETE(exceptionalIds[i].id);
```

Comment:
Please, provide more informative name instead of **a** and check other places in scope of your changes